recipe/meta.yaml

```diff
@@ -1,23 +1,28 @@
 {% set name = "grpc-cpp" %}
-{% set version = "1.29.1" %}
+{% set version = "1.36.4" %}
 
 package:
   name: {{ name | lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/grpc/grpc/archive/v{{ version }}.tar.gz
-  sha256: 0343e6dbde66e9a31c691f2f61e98d79f3584e03a11511fad3f10e3667832a45
+  sha256: 8eb9d86649c4d4a7df790226df28f081b97a62bf12c5c5fe9b5d31a29cd6541a
   patches:
     - aarch64-tcp-header.patch  # [aarch64]
     - force-protoc-executable.patch
 
 build:
-  number: 3
+  number: 1
   string: h{{ PKG_HASH }}_pb{{ protobuf | replace(".*", "")}}_{{ PKG_BUILDNUM }}
   run_exports:
     - {{ pin_subpackage('grpc-cpp', max_pin='x.x') }}
   ignore_run_exports:
+    - abseil-cpp
```

Comment:
abseil-cpp may actually be needed so I may remove this. It was listed in run dependencies in our 1.2 package.